wp/lib/bap_wp/src/precondition.ml

```diff
@@ -1388,3 +1396,192 @@ let construct_pointer_constraint (l_orig : Constr.z3_expr list) (env1 : Env.t)
   in
   let expr = List.fold l ~init:(Bool.mk_true ctx) ~f:gen_constr in
   Constr.mk_goal "pointer_precond" expr |> Constr.mk_constr
+
+(* Gets the target destination of a jump. We do not handle indirect jumps. *)
+let target (jmp : Jmp.t) : Tid.t option =
```

Comment:
I wonder if it would be using this in `visit_jmp` to avoid duplication.

Reply:
I'm confused here. Are you suggesting we use `target` inside of `visit_jmp`?

Reply:
No, just *within* `visit_jmp` to simplify the logic. Not that important.